.github/workflows/fb-e2e-suite.yml

```diff
@@ -302,3 +290,17 @@ jobs:
       sha: ${{ inputs.sha || github.event.pull_request.head.sha ||  'null' }}
       setup_services: '--database ps=8.0,QUERY_SOURCE=slowlog'
       tags_for_tests: '(?=.*@pmm-ps-integration)^(?!.*@not-ui-pipeline)'
+
+  rbac:
+    name: PS UI role based access control
+    uses: ./.github/workflows/runner-e2e-tests-codeceptjs.yml
+    secrets: inherit
+    with:
+      pmm_server_image: ${{ inputs.pmm_server_image || 'perconalab/pmm-server:dev-latest' }}
```

Comment:
is this the default flag for dev-latest in pmm3?

Reply:
Fixed tags for rbac and other pipelines that used old tags.
https://github.com/percona/pmm-qa/actions/runs/9676315706